packages/react/src/index.ts

```diff
@@ -98,8 +98,15 @@ export * from "./components/TableHandles/TableHandleMenu/DefaultButtons/DeleteBu
 export * from "./components/TableHandles/TableHandleMenu/TableHandleMenu.js";
 export * from "./components/TableHandles/TableHandleMenu/TableHandleMenuProps.js";
 
+export * from "./components/Comments/Comment.js";
```

Comment:
can you add a todo for later? want to let @nperez0111 if we want to export these as @blocknote/react/comments